python/mujinplanningclient/handeyecalibrationplanningclient.py

```diff
@@ -13,15 +18,38 @@
 class HandEyeCalibrationPlanningClient(RealtimeRobotPlanningClient):
     """Mujin planning client for the HandEyeCalibration task"""
 
-    tasktype = 'handeyecalibration'
```

Comment:
Actually I think this needs to stay hard-coded to this client type. Please revert this to a class attribute and use it in the `super` call

Reply:
Done.